packages/bench-trail/runner.js

```diff
@@ -0,0 +1,333 @@
+#!/usr/bin/env node --expose-gc
```

Comment:
so many many many tests for this file they didn't fit in this PR, I will try to send it via USB to each maintainer of DataPoint